src/firmware/boards/legacy_dongle_stm32f4/normal.c

```diff
@@ -423,7 +428,7 @@ static void send_camera_packet(const void *packet)
  *
  * \pre The transmit mutex must be held by the caller.
  */
-static void send_drive_packet(const void *packet, const uint8_t *serials)
+static void send_drive_packet(const void *packet, const size_t packet_size)
```

Comment:
Comment references the old parameters. Maybe also give units for packet_size

Reply:
changed to :
```
 * \param[in] packet the drive packet to send to the robot
 * \param[in] packet_size The length of the drive packet, in bytes
```